avida-ts/docker/docker-compose.local.yaml

```diff
@@ -0,0 +1,82 @@
+version: "3"
+
+services:
+  cheqd-node:
+    image: ghcr.io/nymlab/cheqd-node:v2.0.1-arm64
+    ports:
+      - target: 9090
+        published: 9090 # grpc
+      - target: 26656
+        published: 26656 # Tendermint peer-to-peer (P2P) port
+      - target: 26657
+        published: 26657 # Tendermint RPC port
+    volumes:
+      - ./scripts/:/tmp
+    entrypoint: /tmp/cheqd-genesis.sh
+    env_file:
+      - ../local.env
+
+  cheqd-resource-uploader:
+    image: ghcr.io/nymlab/cheqd-node:v2.0.1-arm64
+    ports:
+      - target: 26656
+        published: 23356 # Tendermint peer-to-peer (P2P) port
+      - target: 26657
+        published: 23357 # Tendermint RPC port
+    volumes:
+      - ./scripts/:/tmp/scripts/
+      - ./cheqd-resource-artifacts/:/tmp/cheqd-resource-artifacts
+    entrypoint: /tmp/scripts/cheqd-resource-upload.sh
+    env_file:
+      - ../local.env
+    environment:
+      - NODE=http://cheqd-node:26657
+
+  neutron-node:
+    image: ghrc.io/nymlab/neutron-standalone:v3.0.5-arm64
+    ports:
+      - target: 9090
+        published: 9080 # grpc
+      - target: 26656
+        published: 25556 # Tendermint P2P port
+      - target: 26657
+        published: 25557 # Tendermint RPC port
+    environment:
+      - RUN_BACKGROUND=0
+      - CHAINID=neutron-local-1
+
+  # contract: address with instantiate2 neutron1ev8e7z53nm9ncn8jv5efsuv7k5stn7uyt627qmpk9a98netlua0qumsp9n
+  # deployer: neutron demo_2
+  # salt: ascii avida-local
+  neutron-deployer:
+    image: ghrc.io/nymlab/neutron-standalone:v3.0.5-arm64
+    volumes:
+      - ./../../artifacts/:/tmp/artifacts/
+      - ./scripts/:/tmp/scripts/
+      - ../contract.env:/tmp/contract.env
+    depends_on:
+      - neutron-node
+    ports:
+      - target: 26656
```

Comment:
I assume here the same thing with exposing ports

Reply:
updated in https://github.com/nymlab/AVIDA/pull/6/commits/7ce3c1886296d88d19edcfd650036b81c8dcdf75